flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "disable-external-data-checker": true
```

Comment:
You disabled x-checker here yet you have a x-checker stanza in the manifest? That's not going to work if this is disabled here.